src/app/page.tsx

```diff
@@ -38,39 +39,37 @@ export default async function HomePage() {
 
   return (
     <main>
-      <nav className='items-center px-10 py-2 sticky border-b-[1px] top-0 z-40 w-full bg-white dark:border-b-slate-700 dark:bg-background flex justify-between'>
-        <h1 className='text-xl lg:text-2xl font-extrabold'><a href="/">Steward.</a></h1>
-        <ul className='hidden lg:flex justify-center items-center lg:gap-8'>
-        {routeList.map(({ href, label }: RouteProps) => (
-          <a
-            key={label}
-            href={href}
-            className={buttonVariants({ variant: "ghost" })}
-            >
-            {label}
-          </a>
-        ))}
+      <nav className="sticky top-0 z-40 flex w-full items-center justify-between border-b-[1px] bg-white px-10 py-2 dark:border-b-slate-700 dark:bg-background">
+        <h1 className="text-xl font-extrabold lg:text-2xl">
+          <a href="/">Steward.</a>
+        </h1>
+        <ul className="hidden items-center justify-center lg:flex lg:gap-8">
+          {routeList.map(({ href, label }: RouteProps) => (
+            <a key={label} href={href} className={buttonVariants({ variant: "ghost" })}>
+              {label}
+            </a>
+          ))}
         </ul>
-        <div className='flex lg:gap-x-2'>
+        <div className="flex lg:gap-x-2">
           {isLoggedIn && (
-            <div className='hidden lg:flex mt-[5px]'>
+            <div className="mt-[5px] hidden lg:flex">
               <Button>
-                <SignOutButton></SignOutButton>
+                <Link href="/new-user">Dashboard</Link>
               </Button>
             </div>
           )}
           {!isLoggedIn && (
-            <div className='hidden lg:flex mt-[5px]'>
+            <div className="mt-[5px] hidden lg:flex">
               <Button>
-                <SignInButton afterSignInUrl='/dashboard'></SignInButton>
```

Comment:
I should have caught this during review, but navigating to the dashboard directly bypasses adding a user to our DB. I'm using the new-user page to then forward them to the dashboard once I know they've signed in.